Javascript Library Project

bmuhamood opened the pull request:
PROJECT: LIBRARY
This is the first project of the JavaScript module. It is preceded by the lesson about Objects and the Object Constructor. It Consist of Building an HTML, css and JavaScript library page that store Books Information/Data.

mikenath223 reviewed (changes requested):
### Hello @bmuhamood  👋

**Nice work on this, and clean simple UI implementation also.** 👌

However, let's make a couple of fixes.

- [x] Kindly add a way to toggle a book read status. A user should be able to toggle an added book between a read and not-read status.

- [x] After adding a book and clicking anywhere on the book, not the delete button the delete alert message still shows. As this isn't an expected behavior consider fixing it.

- [ ] OPTIONAL - Consider also deleting a book from the local storage when a user deletes a book because refreshing the page still shows books that have been deleted.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

bmuhamood commented:
Dear Colleagues, 

Thanks for the review. 
- Added checkbox to mark book read or not
- Added a book removed success message
- Fixed linters errors

addod19 reviewed (approved):
## Status: Approved :trophy: :balloon: 

Hi @bmuhamood, great job on this project :smile: Having fixed the toggle between "read/unread" :+1: And also fixing the issue with the delete message appearing when a user clicks on any part of the screen :fireworks: I hereby approve the project :slightly_smiling_face: 

Kindly go ahead and merge into development and continue with your next projects :+1: Do not forget to submit the project completion form :+1: 
_Happy codig_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)
